gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/KnoxShellTable.java

```diff
@@ -86,6 +96,172 @@ public KnoxShellTable value(Comparable<? extends Object> value) {
     return col;
   }
 
+  private Conversions getConversion(Comparable<? extends Object> colIndex) {
+      Conversions type = null;
+      if (colIndex instanceof Double) {
+        type = Conversions.DOUBLE;
+      }
+      else if (colIndex instanceof Integer) {
+        type = Conversions.INTEGER;
+      }
+      else if (colIndex instanceof Float) {
+        type = Conversions.FLOAT;
+      }
+      else if (colIndex instanceof Byte) {
+        type = Conversions.BYTE;
+      }
+      else if (colIndex instanceof Short) {
+        type = Conversions.SHORT;
+      }
+      else if (colIndex instanceof Long) {
+        type = Conversions.LONG;
+      }
+      else {
+          throw new IllegalArgumentException();
```

Comment:
nit: would be good to include some String here as to why

Reply:
Thanks for the review! Will follow up with this on another change.

Reply:
Will address this in KNOX-2060